SparseGrids/tasgridUnitTests.hpp

```diff
@@ -32,9 +32,11 @@
 #define __TASGRID_UNIT_TESTS_HPP
 
 #include "tasgridCLICommon.hpp"
+#include <string>
+#include <vector>
 
-enum UnitTests{
-    unit_none, unit_all, unit_cover, unit_except, unit_api, unit_c
+enum UnitTests {
```

Comment:
Why add the space, there is no need to change this line.

Reply:
Fixed.